src/lib/y2storage/filesystems/blk_filesystem.rb

```diff
@@ -137,6 +137,47 @@ def blk_device_basename
         format(_("(%{basename}\u2026)"), basename: basename)
       end
 
+      # Display name to represent the filesystem
+      #
+      # Only multidevice filesystems have a display name
+      #
+      # @return [String, nil]
+      def display_name
+        return nil unless multidevice?
+
+        textdomain "storage"
```

Comment:
Why this call to textdomain here?

Reply:
After discussing it, we have decided to keep it as it is. Maybe in the future we should start using decorators.